exchanges/deribit/deribit_test.go

```diff
@@ -0,0 +1,2397 @@
+	if err != nil {
+		t.Fatal(err)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d)
+	resp, err = d.GetSettlementHistoryByInstrument(context.Background(), btcPerpInstrument, "settlement", "", 10, time.Now().Add(-time.Hour))
+	assert.NoError(t, err)
+	_, err = d.WSRetrieveSettlementHistoryByInstrument(btcPerpInstrument, "settlement", "", 10, time.Now().Add(-time.Hour))
+	assert.NoError(t, err)
+}
+
+func TestSubmitEdit(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.SubmitEdit(context.Background(), &OrderBuyAndSellParams{OrderID: "incorrectID", Advanced: "", TriggerPrice: 0.001, Price: 100000, Amount: 123})
+	assert.NoError(t, err)
+	_, err = d.WSSubmitEdit(&OrderBuyAndSellParams{
+		OrderID:      "incorrectID",
+		Advanced:     "",
+		TriggerPrice: 0.001,
+		Price:        100000,
+		Amount:       123,
+	})
+	assert.NoError(t, err)
+}
+
+// Combo Books Endpoints
+
+func TestGetComboIDS(t *testing.T) {
+	t.Parallel()
+	_, err := d.GetComboIDS(context.Background(), currencyBTC, "")
+	assert.NoError(t, err)
+	combos, err := d.WSRetrieveComboIDS(currencyBTC, "")
+	assert.NoError(t, err)
+	assert.False(t, len(combos) == 0, "no combo instance found for currency BTC")
+}
+
+func TestGetComboDetails(t *testing.T) {
+	t.Parallel()
+	sleepUntilTradablePairsUpdated()
+	_, err := d.GetComboDetails(context.Background(), futureComboTradablePair.String())
+	assert.NoError(t, err)
+	_, err = d.WSRetrieveComboDetails(futureComboTradablePair.String())
+	assert.NoError(t, err)
+}
+
+func TestGetCombos(t *testing.T) {
+	t.Parallel()
+	_, err := d.GetCombos(context.Background(), currencyBTC)
+	assert.NoError(t, err)
+}
+
+func TestCreateCombo(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, d, canManipulateRealOrders)
+	_, err := d.CreateCombo(context.Background(), []ComboParam{})
+	assert.Falsef(t, err != nil && !errors.Is(errNoArgumentPassed, err), "expecting %v, but found %v", errNoArgumentPassed, err)
+	instruments, err := d.GetEnabledPairs(asset.Futures)
+	if err != nil {
+		t.Skip(err)
+	}
+	if len(instruments) < 2 {
+		t.Skip("no enough instrument found")
+	}
+	_, err = d.CreateCombo(context.Background(), []ComboParam{
+		{
+			InstrumentName: instruments[0].String(),
+			Direction:      "sell",
+		},
+		{
+			InstrumentName: instruments[1].String(),
+			Direction:      "sell",
+			Amount:         1200,
+		},
+	})
+	assert.Falsef(t, err != nil && !errors.Is(errInvalidAmount, err), "expecting %v, but found %v", errInvalidAmount, err)
+	_, err = d.CreateCombo(context.Background(), []ComboParam{
+		{
+			InstrumentName: instruments[0].String(),
+			Amount:         123,
+		},
+		{
+			InstrumentName: instruments[1].String(),
+			Direction:      "sell",
+			Amount:         1200,
+		},
+	})
+	assert.NoError(t, err, "expecting error message 'invalid direction', but found %v", err)
```

Comment:
Error:      	Received unexpected error:
        	            	invalid direction, only 'buy' or 'sell' are supported
        	Test:       	TestCreateCombo
        	Messages:   	expecting error message 'invalid direction', but found invalid direction, only 'buy' or 'sell' are supported

Looks like you need to change to `assert.ErrorIs`